robot/EDA/tests/browser/settings/relationships.robot

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Documentation   Validate relationships tab in EDA Settings
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/RelationshipsSettingsPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+Test Setup      Go to EDA settings tab          Relationships
+
+*** Test Cases ***
+Verify relationship settings can retain values on save
+    [Documentation]         Checks the values of 'Reciprocal Method' and 'Allow auto-created
+    ...                     duplicate relations' are retained after making an update to their values
+    ...                     and click on save button
+    [tags]                    unstable        W-7989099
+    Update reciprocal method value      Reciprocal Method=List Setting
+    Update checkbox value       Allow Auto-Created Duplicate Relations=false
```

Comment:
are these updates/edits in the context of the same UI modal ? Then maybe you can have one keyword covering them both and one keyword to validate them

Reply:
Yes it is in the same window. But I am planning use update checkbox keyword across pages. That's why I kept it as a separate keyword.